src/client/Client.cc

```diff
@@ -2362,6 +2362,7 @@ void Client::handle_client_session(const MConstRef<MClientSession>& m)
 
       session->mds_features = std::move(m->supported_features);
       session->mds_metric_flags = std::move(m->metric_spec.metric_flags);
+      root_squashs = std::move(m->root_squashs);
```

Comment:
`*m` is `const`. This `std::move` will result in a copy so it's superfluous.

Reply:
Okay, will fix it.